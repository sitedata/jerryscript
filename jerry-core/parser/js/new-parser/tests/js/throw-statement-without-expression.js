/* This test contains throw statement parser negative tests. */

throw
exp;
