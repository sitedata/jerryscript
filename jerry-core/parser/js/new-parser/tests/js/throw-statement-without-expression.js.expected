
--- Script parsing start ---


--- Script parsing end ---

Parse error 'Expression expected.' at line: 4 col: 1
